gapipy/cache.py

```diff
@@ -168,3 +175,26 @@ def info(self):
 
     def is_cached(self, key):
         return self._client.exists(self.key_prefix + key)
+
+
+if django_settings:
+    class DjangoCache(BaseCache):
+        def __init__(self, *args, **kwargs):
+            assert django_settings.CACHES is not None, "No CACHES found in django settings!"
+            assert django_settings.CACHES.get('gapi') is not None, "No 'gapi' entry found in settings.CACHES!"
+            self.client = django_cache['gapi']
+
+        def clear(self):
+            self.client.clear()
```

Comment:
Oh, I wonder if gapipy ever calls this `clear` for anything... this clears the whole Django cache, right? 🤔

Reply:
That's why I just moved it all to `['gapi']` and you can use a `KEY_PREFIX` to the `CACHES` entry if you want to namespace them like `RedisClient` does above

Reply:
> I wonder if gapipy ever calls this clear for anything

only in a test ... it doesn't explicitly call `clear` internally, leaves that up to the client user.